awx/main/analytics/analytics_tasks.py

```diff
@@ -0,0 +1,16 @@
+
+# Python
+import logging
+
+# AWX
+from awx.main.analytics.subsystem_metrics import Metrics
+from awx.main.dispatch.publish import task
+from awx.main.dispatch import get_local_queuename
+
+logger = logging.getLogger('awx.main.scheduler')
+
+
+@task(queue=get_local_queuename)
+def send_subsystem_metrics():
+    logger.debug("Sending subsystem metrics.")
+    Metrics().send_metrics()
```

Comment:
I like that the local node stats live in a different key and that this part basically creates a copy on the local node into a different hash & sends to remove nodes.